Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -145,6 +159,25 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   [self sendEventWithName:@"remoteNotificationsRegistered" body:notification.userInfo];
 }
 
+- (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
+{
+  if (self.requestPermissionsResolveBlock == nil) return;
+
+  NSMutableDictionary* notificationTypes = [NSMutableDictionary dictionaryWithObjectsAndKeys:@NO, @"alert", @NO, @"badge", @NO, @"sound", nil];
```

Comment:
Gotcha, yeah I did the second approach but I agree with the shorthand of the first approach! Done